web/src/playground_links_popover.ts

```diff
@@ -1,19 +1,25 @@
 import $ from "jquery";
+import assert from "minimalistic-assert";
+import type {Instance as PopoverInstance, ReferenceElement} from "tippy.js";
 import url_template_lib from "url-template";
 
 import render_playground_links_popover from "../templates/popovers/playground_links_popover.hbs";
 
 import * as blueslip from "./blueslip";
 import * as popover_menus from "./popover_menus";
+import type {RealmPlayground} from "./realm_playground";
 import * as realm_playground from "./realm_playground";
 import * as ui_util from "./ui_util";
 
-let playground_links_popover_instance;
+let playground_links_popover_instance: PopoverInstance;
 
 // Playground_info contains all the data we need to generate a popover of
 // playground links for each code block. The element is the target element
 // to pop off of.
-function toggle_playground_links_popover(element, playground_info) {
+function toggle_playground_links_popover(
+    element: ReferenceElement,
+    playground_info: RealmPlayground[],
+): void {
```

Comment:
With respect to the `oncreate(instance) ` inside `popover_menus.toggle_popover_menu`  should 
```
instance.setContent(
                ui_util.parse_html(render_playground_links_popover({playground_info})),
            );
```
be replaced with
```
playground_links_popover_instance.setContent(
                ui_util.parse_html(render_playground_links_popover({playground_info})),
            );
```
Which one is more fundamentally correct??